### PR TITLE
Changes the elb_mapper to count tutorials rather than GET and POST requests. Eliminates the reducer elb_reducer in favor of using the prebuilt aggregate.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: aws/emr/cdo-logs/script/elb_mapper.rb [hunk 1]
```diff
@@ -1,13 +1,94 @@
...
-    puts "POST" + "\t" + "1"
-  else
-    puts "OTHER" + "\t" + "1"
+  # The line starts with a timestamp of the form YYYY-MM-DDTHH:MM:SS.XXXXXXZ,
+  # from which we extract only the date for aggregation.
+  date = line[0..9]
+
+  tutorials.each do |t|
+    if (line.include? "begin" + "\/" + t) ||
+      (line.include? "begin" + "_" + t)
```
Comment (Hamms) [inline aws/emr/cdo-logs/script/elb_mapper.rb]: This line appears to be missing an `||`

**Comment by Hamms** [comment]
Seems to me an actual regex would be a lot safer than a chain of `includes?`

**Comment by ashercodeorg** [comment]
Is the regex or three includes more performant? In terms of safety, why would one be safer than the other (given that the regex would be an exact match for these three patterns)?

**Comment by Hamms** [comment]
I have no idea which is more performant, but I also don't know what the chances are of having the strings you're looking for show up in unexpected parts of the string you're checking; with a well-formed regex, you can confirm that you're only checking the parts of the string with the expected semantic meaning.

I'm not referring to a regex that just checks the string for the existence of those strings; I'd far prefer a regex that actually parses the string we're looking at.

It's probably not a huge problem, but I would consider any numbers produced by this to be just the tiniest bit fuzzy.

**Comment by Hamms** [comment]
LGTM! (I'd love to see a regex in a later PR, but this definitely looks good enough to get some numbers now)

**Pull request merged**